common4j/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
Does this need to be checked in? probably not

Reply:
Added to gitignore and force pushed to resolve this.